EnglishFilter/sections/content_blocker.txt

```diff
@@ -1597,6 +1597,8 @@ maclife.de,kuechengoetter.de,prad.de,infranken.de,nachrichten.at,np-coburg.de,fr
 ! https://github.com/AdguardTeam/AdguardFilters/issues/47876
 topbusiness24.ru,mydomsam.ru,hubuhu.ru,moysadinfo.ru,mybusinessplus.ru,dukand.ru,mydizayn.ru,polzadom.ru,umnodachnik.ru,delnyesovety.ru,soverhenie.ru,mirinteresa.ru,uskorit.ru,dskyar.ru,biscand.ru,smartid24.ru,stroysaminfo.ru,topsovety.ru,sotsvetiya.ru,imdaily.ru,turizmnaok.ru,zdorovyobrazzhizni.ru,kinomann.ru,techinfolife.ru,dommoysad.ru,rteinfo.ru,psisovety.ru,kvilit.ru,speshit.ru,ollss.ru#@#.ad-placement
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/101665
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=ethereumfaucet.info
```

Comment:
please remove the rules for antiadblock

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/101689/commits/9a0524d3d7648f6df4595fa3614f5f0017cd2d7c